lib.sh

```diff
@@ -12,6 +12,7 @@ HTML=/var/www
 NCDATA=/mnt/ncdata
 SNAPDIR=/var/snap/spreedme
 GPGDIR=/tmp/gpg
+SHA256_DIR=/tmp/shas56
```

Comment:
I am pretty sure, that this should be 
`SHA256_DIR=/tmp/sha256`
But its not an important change.

Reply:
Please add it to the "some improvements PR"